### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by Kwentsir**
In this PR, database and project schema was created.

### CODE REVIEW

File: db/migrate/20220824124921_add_references.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddReferences < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, foreign_key: { to_table: :users }, index: true
+    add_reference :comments, :author, foreign_key: { to_table: :users }, index: true
+    add_reference :likes, :author, foreign_key: { to_table: :users }, index: true
```
Comment (gemmen29) [inline db/migrate/20220824124921_add_references.rb:6]: - [ ] Kindly add a reference to the `post` in the `like` and `comment` tables to define the relation between like and post and between comment and post per this [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Kwentsir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220824124921_add_references.rb (hunk 1 above)
Comment (Kwentsir) [reply db/migrate/20220824124921_add_references.rb:6]: @gemmen29 thank you for your review.

**Review by AnselemOdims — APPROVED** [review]
### Hi @Kwentsir  🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**